app/assets/json/translations/locale-es.json

```diff
@@ -248,7 +248,8 @@
             },
             "subtitle": "¿Cuáles de las siguientes opciones describen su raza y su etnia? De las siguientes categorías, elija todas las opciones que correspondan.",
             "text_input_placeholders": {
-                "demographic_input_placeholder": "Especifique..."
+                "demographic_input_placeholder": "Especifique...",
+                "indigenous_group_placeholder": "Especifique el grupo"
```

Comment:
hm weird this is saying "Specify the group" instead of "Specific group". Side note, did you follow the updated workflow on this of pushing up english then putting the translations in phrase, then pulling down? I guess there's no way for me to really tell from the PR

Reply:
@akegan I didn't! All of these code changes were from before your PR. I'll make sure to follow the guide.

And these are the strings from the paper translations so if they're wrong here they're wrong there too